bin/tachyon-start.sh

```diff
@@ -45,6 +45,14 @@ get_env() {
   . $TACHYON_LIBEXEC_DIR/tachyon-config.sh
 }
 
+run_command() {
+  if [ "$RUN_IN_FOREGROUND" = "true" ]; then
```

Comment:
for this `$RUN_IN_FOREGROUND`, it seems not really initialized in `bin/tachyon-start.sh`?